### ISSUE TITLE
Duration and time to the end widgets seem to be incorrect

### ISSUE THREAD

**Issue by elena-zh** [gnosis/ido-ux#393]
see the image: 2 auctions with almost the same duration show differently filled in circles
![ends.jpg](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/1add3ce0-24b5-4b31-8f7c-315b1001169b)

several auctions show almost empty duration circle, however, time to the end is about 1 hour
![ends in.jpg](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/6e48047a-ca5b-47ce-96d1-aedb3805ddc0)

Duration on the 3 main featured auctions seems to be incorrect as well
![ends duration.jpg](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/e636fa9f-1290-490f-b3a5-2f02cbbbd28a)

**Comment by gabitoesmiapodo** [comment]
Not sure about this issue.

It could be that the auctions changed (Alex changed some stuff on the API servers), but I see no issue on the Featured Auctions right now:

- Auctions `#2` and `#3` are only 1:50h from closing, so the circle is almost full.
- Auction `#1` is 8 days from closing, so the circle shows almost no progress.

![image](https://user-images.githubusercontent.com/4015436/113303564-975cd700-92d7-11eb-92dd-184727e61dd6.png)
![image](https://user-images.githubusercontent.com/4015436/113303602-9f1c7b80-92d7-11eb-84f6-5947d31ded95.png)
![image](https://user-images.githubusercontent.com/4015436/113303749-c6734880-92d7-11eb-9b07-f161ca7bfe92.png)

That said, I think there's an issue on really long auctions (say, 2 years or more), where the timer will show wrong values... as in showing a few hours instead of showing 2 years.

We could create some long auctions to test that.

**Comment by elena-zh** [comment]
@gabitoesmiapodo , 
yes, that's correct. Long auctions (auctionId=17&chainId=4#topAnchor) show an incorrect date: 
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/73e7dc28-1430-48b5-978b-8d31485d7ef2)
